README.md

```diff
@@ -430,3 +434,33 @@ You can try to use a software implementation of OpenGL, called [Mesa](https://gi
 > I run f3d from the command prompt and my Unicode characters are not displayed properly.
 
 Set the codepage to UTF-8, run `chcp 65001`.
+
+# Third Party Copyrights
+
+cxxopts.hpp:
```

Comment:
What about VTK, OCCT, Assimp?

Reply:
Not needed as they are not part of the source of F3D.

Maybe we should add a binary release license part, probably in the release.